src/Text/Pandoc/Writers/AsciiDoc.hs

```diff
@@ -45,6 +47,11 @@ data WriterState = WriterState { defListMarker       :: Text
                                , asciidoctorVariant  :: Bool
                                , inList              :: Bool
                                , hasMath             :: Bool
+                               -- |0 is no table
+                               -- 1 is top level table
+                               -- 2 is a table in a table
+                               , tableNestingLevel   :: Int
+                               , log :: [LogMessage]
```

Comment:
Could we change this to `logMessages` to avoid the need to shadow `log` from prelude?

Reply:
Is this even used?

Reply:
It is not used after all, thanks for catching that, I removed it.